docs/source/getting_started/detailed_tutorial.md

```diff
@@ -58,7 +58,7 @@ which defines the providers and their settings.
 Now let's build and run the Llama Stack config for Ollama.
 
 ```bash
-INFERENCE_MODEL=llama3.2:3b llama stack build --template ollama --image-type venv --run
+INFERENCE_MODEL=llama3.2:3b uv run --with llama-stack llama stack build --template ollama --image-type venv --run
```

Comment:
Since the env is sourced above, the `uv run` shouldn't be needed.

Reply:
Nevermind, I think we are missing a `uv sync` step here, can you try and let us know? 

```bash
uv venv
uv sync
source .venv/bin/activate
llama --help
```

Thanks!

Reply:
@leseb, that worked perfectly. I've reverted the change and added the `uv sync` step. Thanks! :)

Reply:
Is `uv venv --python 3.10` necessary in this step as `uv sync` will spawn a venv anyway. 
I can understand why we would keep it for ensuring a particular python version though.

Reply:
We could also run the `uv sync` alongside the python version: `uv sync --python 3.10`

Reply:
That works best actually nice catch

Reply:
Updated now, thanks! :)